docs/tutorials/gs_continuous_space.ipynb

```diff
@@ -0,0 +1,1210 @@
+{
```

Comment:
<p>It seems you were running this with version 3.4.3.something. </p><p>Can you just say that this tutorial is valid for 3.5 (which still, is relatively old by now) to be sure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>